src/services/leafletWatchHelpers.js

```diff
@@ -20,7 +20,17 @@ angular.module("leaflet-directive")
   to set watching to once and its watch depth.
   */
   var _maybeWatch = function(scope, thingToWatchStr, watchOptions, initCb){
-      return _maybe(scope, '$watch', thingToWatchStr, watchOptions, initCb);
+      var localWatchOptions, watchMethod;
```

Comment:
So I am not sure if I like doWatch being possibly a String. I understand it is less code; but I think explicit `watchOptions.type` or `watchOptions.methodName` is more explicit.  Plus `doWatch` indicates Boolean. If your adamant about it being the way it is lets discuss it.